inspirehep/modules/workflows/tasks/submission.py

```diff
@@ -218,6 +219,11 @@ def _send_robotupload(obj, eng):
             data = obj.extra_data.get(extra_data_key) or {}
         else:
             data = obj.data
+
+        if not current_app.config.get('FEATURE_FLAG_ENABLE_SENDING_REFERENCES_TO_LEGACY'):
+            data = copy(data)
```

Comment:
```deepcopy```?

Reply:
why? I am only removing a top-level key.

Reply:
is not need it, just to have a clean ``data``